jenkins/jobs/cd/community_docker_multi_products.groovy

```diff
@@ -0,0 +1,41 @@
+pipelineJob('community-docker-multi-products') {
+    definition {
+        cpsScm {
+            lightweight(true)
+            scriptPath('jenkins/pipelines/cd/pre-release-community-docker-rocky.groovy')
+            scm {
+                git{
+                    remote {
+                        url('https://github.com/PingCAP-QE/ci.git')
+                    }
+                    branch('main')
+                    extensions {
+                        cloneOptions {
+                            depth(1)
+                            shallow(true)
+                            timeout(5)
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    parameters {
+        stringParam ('RELEASE_BRANCH','','')
+        stringParam ('RELEASE_TAG','','')
+        booleanParam ('FORCE_REBUILD', true, '')
+        booleanParam ('NEED_DEBUG_IMAGE', true, '')
+        booleanParam ('DEBUG_MODE', false, '')
+        stringParam('TIDB_HASH', '', '')
+        stringParam('TIKV_HASH', '', '')
+        stringParam('PD_HASH', '', '')
+        stringParam('TIFLASH_HASH','','')
+        stringParam('NG_MONITORING_HASH','','')
+        stringParam('TIDB_BINLOG_HASH','','')
+        stringParam('TICDC_HASH','','')
+        stringParam('IMAGE_TAG',  '', 'default RELEASE_TAG-rocky-pre')
+        stringParam('HUB_PROJECT', 'qa', '')
+        booleanParam('NO_FAILPOINT', false, '')
```

Comment:
Suggested use of parameters `FAILPOINT` instead of `NO_FAILPOINT` to improve the readability.

Reply:
Same suggestion with @purelind